### ISSUE TITLE
Change to the Comment design

### ISSUE THREAD

**Issue by mascon** [jhoerr/box-csharp-sdk-v2#39]
Box appears to have changed the comment design from files and folders to items.

Does this effect the SDK?

**Comment by jhoerr** [comment]
Yes, this will require some pieces to be reimplemented.  The existing file/folder-centric implementation still appears to work in the meantime.
